### PR TITLE
Case study

### PR DESCRIPTION
**Pull request by maximelucas**
- added `multiorder_laplacian` function
- added first case study notebook

### CODE REVIEW

File: xgi/linalg/matrix.py [hunk 1]
```diff
@@ -280,6 +281,64 @@ def laplacian(H, order=1, rescale_per_node=False, index=False):
...
+        Multiorder Laplacian for synchronization in higher-order networks.
+        Physical Review Research, 2(3), 033410.
+
+    """
+
+    Ls = [laplacian(H, order=i, rescale_per_node=rescale_per_node) for i in orders]
+    Ks = [degree_matrix(H, order=i) for i in orders]
+
+    L_multi = 0
+    for L, K, w in zip(Ls, Ks, weights):
```
Comment (leotrs) [inline xgi/linalg/matrix.py:331]: The documentation says that `weights` can be a single number or a list, but if it is a number then this `zip` will fail.

File: xgi/linalg/matrix.py [hunk 2]
```diff
@@ -280,6 +281,64 @@ def laplacian(H, order=1, rescale_per_node=False, index=False):
...
+        L_multi += L * w / np.mean(K)
+
+    _, rowdict, _ = incidence_matrix(H, index=True)  # get index
+
+    if index:
+        return L_multi, rowdict
+    else:
+        return L_multi
```
Comment (leotrs) [inline xgi/linalg/matrix.py:339]: Perform the computation only if necessay:
```suggestion
    if index:
        _, rowdict, _ = incidence_matrix(H, index=True)
        return L_multi, rowdict
    else:
        return L_multi
```

**Review by leotrs — CHANGES_REQUESTED** [review]
Notebook looks good. There's just one issue about the input types. If you have time, it'd be great to add a test though I know testing matrices is a PITA....

File: xgi/linalg/matrix.py (hunk 1 above)
Comment (maximelucas) [reply xgi/linalg/matrix.py:346]: Done

**Review by leotrs — APPROVED** [review]

**Pull request merged**